scripts/qa_crowbarsetup.sh

```diff
@@ -4860,6 +4860,7 @@ function onadmin_prepare_cloudupgrade_admin_repos_6_to_7
 
     # recreate the SUSE-Cloud Repo with the latest iso
     onadmin_prepare_cloud_repos
+    addcloud7pool
```

Comment:
Can you add a comment that we need Pool repos as the `repocheck` step checks for their presence (as they are required for customer setups)?